src/lib/packet-print.c

```diff
@@ -951,6 +955,9 @@ pgp_sprint_pubkey(const pgp_key_t *key, char *out, size_t outsize)
                        BN_bn2hex(key->key.pubkey.key.rsa.e));
         break;
     case PGP_PKA_EDDSA:
+        cc += snprintf(
```

Comment:
Well.. it was from [this commit](https://github.com/riboseinc/rnp/commit/45c3e1f084854d02f96759ecedb74bc1df675045#diff-fe1bdeb4e3cf464301af5c4b584131adR942) but clang-format reformatted it and looks like I added it.

Shall I remove it?